docs/content/en/configuration.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Configuration
+description: 'How to configure this package.'
+position: 3
+category: Getting Started
+---
+
+## Creating a Base Model
+
+See the [API reference](/api/model-options) for a list of available options.
+
+The first step is to create a base model to define the default options, in order to abstract configuration 
+from your models. It should extend the 
+[Base Model](https://github.com/robsontenorio/vue-api-query/blob/master/src/Model.js) of [vue-api-query](https://github.com/robsontenorio/vue-api-query).
+
+The base model must implement two methods:
+- `baseURL` - The base url of your REST API.
+- `request` - The default request method.
+
+Let's create a new file `Model.js` in `models` directory:
+
+```js{}[~/models/Model.js]
+import { Model as BaseModel } from 'vue-api-query'
+
+export default class Model extends BaseModel {
+
+  // Define a base url for a REST API
+  baseURL() {
+    return 'http://my-api.com'
+  }
+
+  // Implement a default request method
+  request(config) {
+    return this.$http.request(config)
+  }
+}
+```
+
+<alert type="info">`$http` property is the HTTP Client Instance configured in [Installation](/installation) section.</alert>
```

Comment:
Is the tag necessary here? I would like some better alert-box mechanism too, but if the file is called .md you should use markdown syntax.

How about using `>`? I admit the default coloring isn't very good in GitHub.

\> This has its own box
⬇
> This has its own box